README_dev.md

```diff
@@ -35,51 +63,103 @@ which python
 whereis python
 ```
 
-Then create a poetry virtual environment using that version of Python:
+Then create a Poetry virtual environment using the same version of Python:
+
 ```bash
 poetry env use $(which python)
 ```
 
-Then install the dev dependencies:
+Install the dev dependencies:
+
 ```bash
 make dev_env
 ```
 
-### Virtualenv
-***
-To use virtualenv to set up the project, first you need to create a virtualenv environment folder in the root of the repo:
+#### Virtualenv
+
+First, create a virtualenv environment in the root of the repo:
+
 ```bash
 python -m venv .venv
 ```
 
 Then activate it:
+
 ```bash
 source .venv/bin/activate
 ```
 
-To install the dependencies from any of the requirements.txt files in the repo, run:
+</details>
+
+First, install dependencies:
+
 ```bash
-pip install -r <path/to/requirements.txt>
+pip install -r ./requirements.txt
+pip install toml mage-ai
 ```
 
-Then, to install the dev-dependencies from the `pyproject.toml` file, run:
+Install additional dev dependencies from `pyproject.toml`:
+
 ```bash
 pip install $(python -c "import toml; print(' '.join(toml.load('pyproject.toml')['tool']['poetry']['group']['dev']['dependencies'].keys()))" | tr '\n' ' ')
 ```
 
-This command just uses Python and the `toml` library to output the dev dependencies from the `pyproject.toml` as a space-delimited list, and passes that output to the `pip install` command.
-## Git Hooks
+<sup>The above command uses the `toml` library to output the dev dependencies from the `pyproject.toml` as a space-delimited list, and passes that output to the `pip install` command.</sup>
+
+Then, start the dev server:
+
+```bash
+python mage_ai/server/server.py
```

Comment:
This command doesn't run Mage in docker.
If you just want to run the server only, you can use `./scripts/start.sh default_repo`

Reply:
Updated

Reply:
@mattppal @wangxiaoyou1993 

I think there's a misunderstanding here. The note to start the server only with the script should be placed at Docker section, the section you are quoting is to setup a local server, hence the Python command. I'll make the changes.